src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -180,8 +180,14 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 },
 
                 get: el => LocationAccessorsInstrumentation.isLocationWrapper(el) ? el.href : PropertyAccessorsInstrumentation._getUrlAttr(el, 'href'),
-                set: (el, value) => LocationAccessorsInstrumentation.isLocationWrapper(el) ?
-                                    el.href = destLocation.resolveUrl(value, document) : el.setAttribute('href', value)
+                set: (el, value) => {
+                    if (LocationAccessorsInstrumentation.isLocationWrapper(el))
+                        el.href = destLocation.resolveUrl(value, document);
+                    else
+                        el.setAttribute('href', value);
+
+                    return void 0;
```

Comment:
more precisely:
if `el.href=<value>` is called then we should return  <value>.
else retrun `void 0`.

![tmp](https://cloud.githubusercontent.com/assets/4133518/15318623/911fc6c8-1c2f-11e6-97ec-392695ca9c5f.png)